### PR TITLE
Sync with v.1.0.0 release of metadatascheme

### PR DESCRIPTION
**Pull request by PatrickDekkerHealthRI**
- Replaced shacls, by URL to github datamodel repo.
- Update readme, with new instructions
- Update config -> banner.html to point to right location for documentation.
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,29 +8,26 @@ The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthr
 
 ## Development
 
-Run `docker compose build && docker compose up` and a validator will be exposed at <http://localhost:8080/shacl/fdp/upload>.
+Run `docker compose build && docker compose up` and a validator will be exposed at <http://localhost:8080/shacl/healthri/upload>.
 
-Currently there are two profiles. The basic profile will validate against the shapes for `dcat:Resource`,
-the full profile will validate against all shapes.
+Currently there are two profiles v.1.0.0 and development. The v1.0.0. profile will validate against the v1.0.0 release shacls,
```
Comment (Markus92) [inline README.md:13]:
```suggestion
Currently there are two profiles: v.1.0.0 and development. The v1.0.0. profile will validate against the v1.0.0 release SHACLs, 
```

File: README.md [hunk 2]
```diff
@@ -8,29 +8,26 @@ The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthr
 
 ## Development
 
-Run `docker compose build && docker compose up` and a validator will be exposed at <http://localhost:8080/shacl/fdp/upload>.
+Run `docker compose build && docker compose up` and a validator will be exposed at <http://localhost:8080/shacl/healthri/upload>.
 
-Currently there are two profiles. The basic profile will validate against the shapes for `dcat:Resource`,
-the full profile will validate against all shapes.
+Currently there are two profiles v.1.0.0 and development. The v1.0.0. profile will validate against the v1.0.0 release shacls, 
+development will validate against development, note: development can contains errors, so use with care!.
```
Comment (Markus92) [inline README.md:14]:
```suggestion
development will validate against the latest development version. Note: development can contains errors, so use with care!
```

File: README.md [hunk 3]
```diff
@@ -8,29 +8,26 @@ The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthr
...
+The shacles are retrieved from github datamodel repository.
```
Comment (Markus92) [inline README.md:15]:
```suggestion
The SHACLs are retrieved directly from the github datamodel repository.
```

File: validator/resources/healthri/config.properties [hunk 4]
```diff
@@ -1,11 +1,37 @@
...
+validator.bannerHtml = <div><div style=display:table><div style=display:table-row><div style=display:table-cell><h1>Health-RI RDF Validator</h1></div></div><div style=display:table-row><div style=display:table-cell;padding-top:20px><p>This service allows you to validate RDF content against <a href=https://www.w3.org/TR/shacl/ >SHACL shapes</a>. The shapes conform to the <a href=https://github.com/Health-RI/health-ri-metadata/>Health-RI core metadata</a> shapes, as produced in Plateau 1.<p>There might still be issues with these shapes. Please contact <a href=https://www.health-ri.nl>Health-RI</a> by opening an issue at our <a href=https://raw.githubusercontent.com/Health-RI/metadata-shacl-validation/issues/new>Github repository</a>.<p>As soon as any additional Health-RI petal shapes are finished, they will be added to this validator.</div></div></div><hr></div>
```
Comment (Markus92) [inline validator/resources/healthri/config.properties:37]:
```suggestion
validator.bannerHtml = <div><div style=display:table><div style=display:table-row><div style=display:table-cell><h1>Health-RI RDF Validator</h1></div></div><div style=display:table-row><div style=display:table-cell;padding-top:20px><p>This service allows you to validate RDF content against <a href=https://www.w3.org/TR/shacl/ >SHACL shapes</a>. The shapes conform to the <a href=https://github.com/Health-RI/health-ri-metadata/>Health-RI core metadata</a> shapes, as produced in Plateau 1.<p>There might still be issues with these shapes. Please contact <a href=https://www.health-ri.nl>Health-RI</a> by opening an issue at our <a href=https://github.com/Health-RI/metadata-shacl-validation/issues/new>Github repository</a>.<p>As soon as any additional Health-RI petal shapes are finished, they will be added to this validator.</div></div></div><hr></div>
```
Dead link

File: README.md [hunk 5]
```diff
@@ -8,29 +8,26 @@ The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthr
...
-The shapes can be placed in the *shapes* folder or a subfolder thereof.
-
-To add the new profile, edit *config.properties* file. At the first line, add a new profile to the
+To add a new version/profile, edit *config.properties* file. At the first line, add a new profile to the
 `validator.type` line. Then, add two lines: `validator.typeLabel.<your profile> = <profile description>`
-is the title that would appear in the drop down menu.
-`validator.shaclFile.<yourprofile> = shapes/<your file>, shapes/<your file 2>` you can add SHACL shapes,
-comma-separated.
+this is the title that would appear in the drop down menu.
+`validator.shaclFile.<your profile>.remote.<#>.url = http://..... and
```
Comment (Markus92) [inline README.md:27]:
```suggestion
`validator.shaclFile.<your profile>.remote.<#>.url = http://.....` and
 
```

File: README.md [hunk 6]
```diff
@@ -8,29 +8,26 @@ The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthr
...
-
-To add the new profile, edit *config.properties* file. At the first line, add a new profile to the
+To add a new version/profile, edit *config.properties* file. At the first line, add a new profile to the
 `validator.type` line. Then, add two lines: `validator.typeLabel.<your profile> = <profile description>`
-is the title that would appear in the drop down menu.
-`validator.shaclFile.<yourprofile> = shapes/<your file>, shapes/<your file 2>` you can add SHACL shapes,
-comma-separated.
+this is the title that would appear in the drop down menu.
+`validator.shaclFile.<your profile>.remote.<#>.url = http://..... and 
+`validator.shaclFile.<your profile>.remote.<#>.type = text/turtle
```
Comment (Markus92) [inline README.md:28]:
```suggestion
`validator.shaclFile.<your profile>.remote.<#>.type = text/turtle`
```

**Review by Markus92 — CHANGES_REQUESTED** [review]
Great work! I made some small textual suggestions to the readme.

File: banner.html [hunk 7]
```diff
@@ -11,7 +11,7 @@ <h1>Health-RI RDF Validator</h1>
                 <p> This service allows you to validate RDF content against <a href="https://www.w3.org/TR/shacl/">SHACL
                         shapes</a>.
                     The shapes conform to the <a
-                        href="https://health-ri.atlassian.net/wiki/spaces/FSD/pages/121110529/Core+Metadata+Schema+Specification">Health-RI
+                        href=https://github.com/Health-RI/health-ri-metadata/>Health-RI
```
Comment (kburger) [inline banner.html:14]: Could we link this to a `latest` release? Also depends on what we end up putting in the description of the release.

File: validator/resources/healthri/config.properties [hunk 8]
```diff
@@ -1,11 +1,37 @@
-validator.type = review
+validator.type = v1.0.0, development
```
Comment (kburger) [inline validator/resources/healthri/config.properties:1]: Do we dare to add a `latest` tag here, linking to https://github.com/Health-RI/health-ri-metadata/releases/latest? That would assume that the shapes are attached as artifacts to the release.

File: banner.html (hunk 7 above)
Comment (PatrickDekkerHealthRI) [reply banner.html:14]: will delete this banner file. It not used (i figured out when updating the link) The banner is located in the config. file.

File: README.md [hunk 9]
```diff
@@ -4,19 +4,19 @@
 
 This repository is used to provide SHACL shapes to verify Health-RI and FAIR Data Point RDF files.
 
-The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthri/upload>.
+The validator itself is available at <https://www.itb.ec.europa.eu/SHACL/healthri/upload>.
```
Comment (kburger) [inline README.md:7]: The uppercase link seems to not redirect properly to the lowercase version. I'd suggest to revert back to the lowercase one.

```suggestion
The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthri/upload>.
```

File: validator/resources/healthri/config.properties (hunk 8 above)
Comment (PatrickDekkerHealthRI) [reply validator/resources/healthri/config.properties:1]: I tried that (and would be the best solution) but 'latest; only points to latest release zip file.

File: README.md (hunk 9 above)
Comment (PatrickDekkerHealthRI) [reply README.md:7]: Yes you're right... i was a overactive when replacing 'shacl' by 'SHACL' as mark asked...

Comment (Markus92) [inline README.md]:
```suggestion
The validator itself is available at <https://www.itb.ec.europa.eu/shacl/healthri/upload>.
```
all caps 404s

**Review by Markus92 — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (PatrickDekkerHealthRI) [reply README.md]: fixed in last commit

**Review by Markus92 — APPROVED** [review]
LGTM!

**Pull request merged**